### ISSUE TITLE
Secure Html renderer wrong window.cookiesConfig

### ISSUE THREAD

**Issue by minhluan259** [magento/magento2#31385]
### Preconditions (*)

Magento 2.4.1

### Steps to reproduce (*)

vendor/magento/module-cookie/view/base/templates/html/cookie.phtml

### Expected result (*)

```
<script>
    window.cookiesConfig = window.cookiesConfig || {};
    window.cookiesConfig.secure = true;
</script>
```

### Actual result (*)

```
<script>true</script>
```

---
Please provide [Severity](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#backlog) assessment for the Issue as Reporter. This information will help during Confirmation and Issue triage processes.

- [ ] Severity: **S0** _- Affects critical data or functionality and leaves users without workaround._
- [ ] Severity: **S1** _- Affects critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S2** _- Affects non-critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S3** _- Affects non-critical data or functionality and does not force users to employ a workaround._
- [ ] Severity: **S4** _- Affects aesthetics, professional look and feel, “quality” or “usability”._

**Comment by mrtuvn** [comment]
This issue already fixed in 2.4-develop branch and already delivered to next update 2.4.2 (reported by @bubasuma)
CC: minhluan259

**Comment by ihor-sviziev** [comment]
Hi @minhluan259 ,
This issue was already fixed in 2.4-develop branch in https://github.com/magento/magento2/commit/d04e4d505aced753aedadf1f3593ef5420825857.
I’m closing this issue as not relevant anymore
